pages/index.html

```diff
@@ -4,9 +4,56 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
+  <link href="../styles/index.css" rel="stylesheet">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+  <link rel="apple-touch-icon" sizes="180x180" href="../images/favicon/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="../images/favicon/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="../images/favicon/favicon-16x16.png">
+  <link rel="manifest" href="/site.webmanifest">
+  <title>Sarah Sanborn</title>
 </head>
-<body>
+<body class="body-index">
+  <header class="header-index">
+    <nav>
+      <a href="index.html"><i class="fa fa-home fa-3x"></i></a>
+      <ul>
+        <li><a href="about.html">About</a></li>
+        <li><a href="portfolio.html">Portfolio</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main class="main-index">
+    <div class="hero-image">
+      <!-- Background image by Sam Shultz at https://unsplash.com/photos/djSi86zsRiw -->
+      <section class="hero-text">
+        <h2 class="hero-title">Hi, it's me Sarah</h2>
```

Comment:
In semantic HTML, heading tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/).  There is no `<h1>` tag in this document, so I'd recommend making this an `<h1>` tag rather than h2.